### PR TITLE
added discord link to get-involved page and footer 4th column

### PR DESCRIPTION
**Pull request by LordSimal**
Since Font Awesome 4.6.3 is used there was no Discord Icon present in that font.
Therefore I added the Discord SVG and changed the color to match all other icons.

This also adds the discord link to the 4th footer column.

### CODE REVIEW

File: config/site.php [hunk 1]
```diff
@@ -137,6 +137,11 @@
                         'options' => ['target' => '_blank'],
                         'title' => 'Slack',
                     ],
+                    'discord' => [
+                        'url' => 'https://discord.gg/A2VHpxeq',
```
Comment (ADmad) [inline config/site.php:141]:
```suggestion
                        'url' => 'https://discord.gg/k4trEMPebj',
```

Changed temp invite link to non-expiring link.

File: src/Template/Element/get-involved/get-help.ctp [hunk 2]
```diff
@@ -22,6 +22,19 @@ use Cake\Core\Configure;
 			?>
 		</div>
 	</div>
+    <div class="col-sm-12 p0">
```
Comment (ADmad) [inline src/Template/Element/get-involved/get-help.ctp:25]: Indentation looks off here.

File: src/Template/Element/get-involved/get-help.ctp [hunk 3]
```diff
@@ -22,6 +22,19 @@ use Cake\Core\Configure;
...
+        </div>
+        <div class="col-sm-11 p0">
+            <?= $this->Html->tag(
```
Comment (ADmad) [inline src/Template/Element/get-involved/get-help.ctp:30]: Can put the `<p>` tag directly in HTML instead of using the helper.

Comment (LordSimal) [reply src/Template/Element/get-involved/get-help.ctp]: I allowed myself to transform all other Html Helper p tags to "normal" p tags as well 😉

**Comment by steinkel** [comment]
I see only minor cs issues here, would you like to fix it or shall I pass a cs-fixer and merge @LordSimal ?

Thanks!

**Comment by LordSimal** [comment]
well my last comment was just to show my confusion of how the Indentation is being defined in the .editorconfig and how it should be. Don't know what the correct way is now.

**Comment by steinkel** [comment]
I'll merge and reformat it later

**Pull request merged**